src/components/shared/accountsTable/index.js

```diff
@@ -0,0 +1,135 @@
+        </BoxHeader>
+        {
+          accounts.error
+            ? (
+              <BoxContent>
+                <BoxEmptyState>
+                  <Illustration name="emptyWallet" />
+                  <h3>{emptyStateMessage || `${accounts.error}`}</h3>
+                </BoxEmptyState>
+              </BoxContent>
+            )
+            : (
+              <React.Fragment>
+                <BoxContent className={styles.content}>
+                  <Table
+                    getRowLink={account => `${routes.accounts.path}/${account.address}`}
+                    data={accounts.data}
+                    columns={[
+                      {
+                        header: t('Rank'),
+                        className: `${grid['col-xs-1']} ${grid['col-md-1']}`,
+                        id: 'rank',
+                        getValue: () => (<span className={styles.counter} />),
+                      },
+                      {
+                        header: t('Address'),
+                        className: `${grid['col-xs-3']} ${grid['col-md-5']}`,
+                        id: 'address',
+                        getValue: account => (
+                          <AccountVisualWithAddress
+                            address={account.address}
+                            isMediumViewPort={isMediumViewPort}
+                            transactionSubject="address"
+                            showBookmarkedAddress
+                          />
+                        ),
+                      },
+                      {
+                        header: t('Balance'),
+                        className: `${grid['col-xs-3']} ${grid['col-md-3']}`,
+                        id: 'balance',
+                        getValue: account => (
+                          <LiskAmount val={account.balance} roundTo={0} token={tokenMap.LSK.key} />
+                        ),
+                      },
+                      {
+                        header: t('Supply'),
+                        className: `${grid['col-xs-2']} ${grid['col-md-1']}`,
+                        id: 'supply',
+                        getValue: (account) => {
+                          const amount = new BigNumber(account.balance / supply * 100);
+                          return <span>{`${amount.toFormat(2)} %`}</span>;
+                        },
+                      },
+                      {
+                        header: t('Owner'),
+                        className: `${grid['col-xs-3']} ${grid['col-md-2']}`,
+                        id: 'owner',
+                        getValue: (account) => {
+                          const delegateUsername = account.delegate ? account.delegate.username : '';
+                          const text = account.knowledge
+                            ? `${account.knowledge.owner} ${account.knowledge.description}`
+                            : delegateUsername;
+                          return (<span>{text}</span>);
```

Comment:
💅 Is the span needed? IMO less code is usually better
```suggestion
                          return text;
```